cvat/apps/lambda_manager/serializers.py

```diff
@@ -5,30 +5,35 @@
 from drf_spectacular.utils import extend_schema_serializer
 from rest_framework import serializers
 
+class SkeletonLabelMappingEntrySerializer(serializers.DictField):
+    name = serializers.CharField()
+    attributes = serializers.DictField(child=serializers.CharField(), required=False)
```

Comment:
I would suggest using a dedicated class as the child, e.g.:

```
class AttributeMappingEntrySerializer(serializers.Serializer):
    name = serializers.CharField()
```

That way, it can be extended if we decide to add more attribute-specific mapping information in the future.

Reply:
We can rewrite it in the future if it will be necessary

Reply:
So, now are we ready to break it?
Do we need ``labels_v2`` in this case? Shall we remove previous ``attributes`` and ``labels`` from response body?

Reply:
What do you mean? Isn't this a new field?

Reply:
So you're saying that attribute mapping was already supported, it just wasn't reflected in the serializer? In that case yeah, we shouldn't change it.

Reply:
Models body from the server before contained:
```
{
    attributes: [string1, string2, string3],
    labels: [string1, string2, string3],
}
```

After this patch server returns 
```
type MLModelLabel = {
    attributes: [string1, string2, string3],
    labels: [string1, string2, string3],
    labels_v2: {
        name: string;
        type: ShapeType | 'unknown';
        attributes: {
            name: string;
            values: string[];
            input_type: 'select' | 'number' | 'checkbox' | 'radio' | 'text';
        }[];
        sublabels?: MLModelLabel[];
        svg?: string,
    }[]
}
```